.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,28 @@
+on:
+    push:
+      paths:
+        - paper/**
+        - .github/workflows/draft-pdf.yml
+
+jobs:
+  paper:
+    runs-on: ubuntu-latest
+    name: Paper Draft
+    steps:
+      - name: Checkout
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)